rpc/executionapi.go

```diff
@@ -115,6 +116,18 @@ func (ec *ExecutionClient) GetClientVersion(ctx context.Context) (string, error)
 	return result, err
 }
 
+func (ec *ExecutionClient) GetAdminPeers(ctx context.Context) ([]*p2p.PeerInfo, error) {
+	var result []*p2p.PeerInfo
+	err := ec.rpcClient.CallContext(ctx, &result, "admin_peers")
```

Comment:
that's a quite heavy namespace we require here (`admin`).
that namespace might not be available for external EL nodes, so the error produced here should not bubble up and make the whole ExecutionClient unusable.
maybe catch these errors in the `updateNodePeers` function and just set a empty peerlist/peerid if a error is thrown?